src/App.tsx

```diff
@@ -1,61 +1,102 @@
+import React, { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
+import usersFromServer from './api/users';
```

Comment:
Ensure that `usersFromServer` is correctly imported and contains the expected data structure. If `usersFromServer` is not an array or does not have `id` and `name` properties, this will cause errors when rendering the user options.